test/builtin/numbers/test_nintegrate.py

```diff
@@ -2,17 +2,18 @@
 """
 NIntegrate[] tests
 
-This also
 """
 import importlib
 import pytest
 from test.helper import evaluate
 
 # From How to check if a Python module exists without importing it:
 # https://stackoverflow.com/a/14050282/546218
-scipy_integrate = importlib.util.find_spec("scipy.integrate")
+scipy_integrate_module = None
+if importlib.util.find_spec("scipy") is not None:
+    scipy_integrate_module = importlib.util.find_spec("scipy.integrate")
```

Comment:
Here we could also use the ``check_requires_list`` instead of the three lines.

Reply:
Please make the change.